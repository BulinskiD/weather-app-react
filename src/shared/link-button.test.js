import React from 'react';
import LinkButton from './link-button';
import {shallow} from "enzyme";


describe('Link button', ()=>{

    it('should render correctly with given props', ()=>{
        const path= "test";
        const component = shallow(<LinkButton path={path}>Text</LinkButton>);
        expect(component.find("Link").props().children).toBe("Text");
        expect(component.find("Link").props().to).toBe("test");
        expect(component).toMatchSnapshot();
    });
});
