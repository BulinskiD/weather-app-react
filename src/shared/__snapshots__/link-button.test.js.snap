// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link button should render correctly with given props 1`] = `
<Link
  className="btn btn-primary offset-lg-10 col-lg-2 offset-0 col-6 bg-light text-info"
  to="test"
>
  Text
</Link>
`;
